src/ocean/time/WallClock.d

```diff
@@ -167,16 +167,3 @@ struct WallClock
                 return fromDate(date) + TimeSpan(mod);
         }
 }
-
-
-version (Posix)
-{
-    version (darwin) {}
-    else
-    {
-        static this()
-        {
-            tzset();
```

Comment:
Mmm, this looks a little scary. Even when the man says that is called by other functions using timezone data, the user can potentially use `tzname` and assume it is properly initialized without calling this function. I know that chances of breakage are very slim (and I'd really like to see this static ctor gone!), but technically this is a breaking change so I would delay it for v4, so users get a big notice that from now on they need to call it themselves (if needed).